src/fsharp/FSharp.Build/Microsoft.FSharp.NetSdk.targets

```diff
@@ -53,7 +53,7 @@ WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and
   </PropertyGroup>
 
   <PropertyGroup Condition=" '$(DisableImplicitSystemValueTupleReference)' != 'true' ">
-    <_FrameworkNeedsValueTupleReference Condition=" $(TargetFramework.StartsWith(netcoreapp1.)) or $(TargetFramework.StartsWith(netstandard1.)) ">true</_FrameworkNeedsValueTupleReference>
+    <_FrameworkNeedsValueTupleReference Condition=" ('$(TargetFrameworkIdentifier)' == '.NETStandard' or '$(TargetFrameworkIdentifier)' == '.NETCoreApp') and $(TargetFrameworkVersion.CompareTo(&quot;v2.0&quot;)) &lt; 0 ">true</_FrameworkNeedsValueTupleReference>
     <_FrameworkNeedsValueTupleReference Condition=" '$(TargetFramework)' == 'net40' or '$(TargetFramework)' == 'net45' or '$(TargetFramework)' == 'net46' or '$(TargetFramework)' == 'net461' or '$(TargetFramework)' == 'net462' or '$(TargetFramework)' == 'net47' ">true</_FrameworkNeedsValueTupleReference>
```

Comment:
Can you please change this line as well - It sounds like basically anything higher than 4.0